### ISSUE TITLE
unexpected keyword argument 'use_dualstack_endpoint'

### ISSUE THREAD

**Issue by awhillas** [mintel/pytest-localstack#52]
Getting this error:
```
self = <botocore.client.ClientEndpointBridge object at 0x7fd42b40d7f0>, service_name = 's3', region_name = 'ap-southeast-1'
endpoint_url = None, is_secure = True

    def resolve(self, service_name, region_name=None, endpoint_url=None,
                is_secure=True):
        region_name = self._check_default_region(service_name, region_name)
        use_dualstack_endpoint = self._resolve_use_dualstack_endpoint(
            service_name)
        use_fips_endpoint = self._resolve_endpoint_variant_config_var(
            'use_fips_endpoint'
        )
>       resolved = self.endpoint_resolver.construct_endpoint(
            service_name, region_name,
            use_dualstack_endpoint=use_dualstack_endpoint,
            use_fips_endpoint=use_fips_endpoint,
        )
E       TypeError: construct_endpoint() got an unexpected keyword argument 'use_dualstack_endpoint'
```
Python 3.9.10
boto3 '1.20.23'
localstack '0.14.0.5'
pytest '6.2.5'

Botocore has changed its API?

**Comment by tiyberius** [comment]
I'm getting this error too. Is this project still being updated?

**Comment by jtdoepke** [comment]
Should be fixed by #51 in v0.5.0.
